docs/workflows.rst

```diff
@@ -357,14 +357,18 @@ tCompCor, aCompCor, Framewise Displacement, 6 motion parameters, DVARS, and, if
 the ``--use-aroma`` flag is enabled, the noise components identified by ICA-AROMA
 (those to be removed by the "aggressive" denoising strategy).
 
-*Note*: Confounds for performing *non*-aggressive denoising cannot be generated in FMRIPREP.
-If the ``--use-aroma`` flag is passed to FMRIPREP, the MELODIC mix and noise component indices will
-be generated, and non-aggressive denoising may be performed with ``fsl_regfilt``, *e.g.*::
+*Note*: Confounds for performing *non*-aggressive AROMA denoising cannot be generated in FMRIPREP.
+However, for conveninece FMRIPREP produces *non*-aggressive denoised 4D NIFTI files in the MNI space
```

Comment:
I think maybe we should revisit the first line here; since confounds for performing non-aggressive denoising are the same as aggressive (it's the denoising process that's at issue), we don't need to say that FMRIPREP is unable to generate them. We could explain that non-aggressive denoising is its own process, not something that can be done by including nuisance regressors in a GLM. Or just leave it out, e.g.:

```RST
For convenience, FMRIPREP produces *non*-aggressively denoised BOLD series in the MNI space
```

In any event, "convenience" is misspelled and [NIfTI](https://nifti.nimh.nih.gov/) should have a lowercase f.